pydantic/_internal/_model_construction.py

```diff
@@ -139,9 +139,11 @@ def complete_model_class(
     except PydanticUndefinedAnnotation as e:
         if raise_errors:
             raise
-        cls.__pydantic_validator__ = MockValidator(  # type: ignore[assignment]
-            f'`{name}` is not fully defined, you should define `{e}`, then call `{name}.model_rebuild()`'
-        )
+        warning_string = f'`{name}` is not fully defined, you should define `{e}`, then call `{name}.model_rebuild()`'
+        # print(cls.__config__)
```

Comment:
Vestige of development, remove:
```suggested
        warning_string = f'`{name}` is not fully defined, you should define `{e}`, then call `{name}.model_rebuild()`'
```

Reply:
why?

Reply:
^ I attempted to leave a code suggestion, but it didn't go. What my comment **meant** to say is >> "remove the print statement"